components/cover/basic/src/index.scss

```diff
@@ -0,0 +1,70 @@
+@import '~@schibstedspain/theme-basic/lib/index';
```

Comment:
IMO controlling warnings and avoiding them to display is a good practice and with tiny cost. I usually do it.